scripts/pipelines/azure-devops/templates/eks/eks-pipeline.cfg

```
@@ -82,9 +78,24 @@ function addPipelineVariables {
     cd "${localDirectory}"
     organization=$(git config --get remote.origin.url)
     organization=$(echo $organization | cut -d/ -f4)
+    
     echo -e "${green}Creating a PAT with necessary permissions..."
     patToken=$(az rest --method post --uri "https://vssps.dev.azure.com/$organization/_apis/tokens/pats?api-version=7.1-preview.1" --resource "https://management.core.windows.net/" --body '{ "displayName": "CreateEKSVariablesToken", "validTo": "2099-02-25T11:44:36.1966667Z", "scope": "vso.variablegroups_manage vso.build_execute" }')
     patToken=$(echo "$patToken" | python -c  "import sys, json; print(json.load(sys.stdin)['patToken']['token'])")
     az pipelines variable create --name "pat" --pipeline-name ${pipelineName} --value $patToken
+    addAWSCredentialsVariableGroup
 
-}
\ No newline at end of file
+}
+
+function addAWSCredentialsVariableGroup {
+    groupid=$(az pipelines variable-group list --group-name "AWS_Credentials" --query '[0].id' -o  json)
+    if test -z $groupid
+    then
+        echo -e "${green}Creating AWS Credentials variable group..."
+        az pipelines variable-group create --name "AWS_Credentials" --variable "REGION"="$awsRegion" --authorize true > ./tmp_group_var
+        new_group_id=$(cat  ./tmp_group_var | grep '"id"' | cut -d: -f2 | cut -d, -f1 | tr -d \")
+        rm ./tmp_group_var
+        az pipelines variable-group variable create --group-id $new_group_id --name "SECRET_KEY" --value "$awsSecretAccessKey" --secret true
+        az pipelines variable-group variable create --group-id $new_group_id --name "ACCESS_KEY" --value "$awsAccessKey" --secret true 
+    fi
```

This should somehow informatively fail is the variable group does not exist and the AWS flags are not provided.

---

Done